sos/report/plugins/foreman.py

```diff
@@ -82,53 +85,29 @@ def setup(self):
         # Allow limiting these
         self.add_copy_spec([
             "/etc/foreman/",
-            "/etc/foreman-proxy/",
             "/etc/sysconfig/foreman",
             "/etc/sysconfig/dynflowd",
-            "/etc/smart_proxy_dynflow_core/settings.yml",
             "/etc/default/foreman",
-            "/etc/foreman-installer/",
             "/var/log/foreman/dynflow_executor*log*",
             "/var/log/foreman/dynflow_executor*.output*",
             "/var/log/foreman/apipie_cache*.log*",
             "/var/log/foreman/cron*.log*",
             "/var/log/foreman/db_migrate*log*",
             "/var/log/foreman/db_seed*log*",
             "/var/log/foreman/production.log[.-]*",
-            "/var/log/foreman-proxy/cron*log*",
-            "/var/log/foreman-proxy/migrate_settings*log*",
-            "/var/log/foreman-proxy/proxy*log*",
-            "/var/log/foreman-proxy/smart_proxy_dynflow_core*log*",
-            "/var/log/foreman-selinux-install.log",
             "/var/log/foreman-proxy-certs-generate*",
```

Comment:
Yeah, we have somehow merged katello plugin into foreman one. Since katello plugin would be just few logfiles, I *think*.

Anyway, we can also separate katello plugin, if we figure out what precisely falls into what plugins scope.

Reply:
I think that Foreman plugins belong in the `foreman` plugin. Foreman Proxy plugins belong in the `foreman_proxy`. However, we also have `foreman-installer` (technically the `foreman-installer-katello` subpackage) and do ship the `foreman-proxy-certs-generate` command with that. AFAIK today that package is installed on both what we call downstream Satellite and Capsule deployments even if should only be used on Satellite installations (today).